tests/test_selafin.cpp

```diff
@@ -115,6 +115,10 @@ TEST( MeshSLFTest, MalpassetGeometry )
   value = getValue( ds, 9571 );
   EXPECT_DOUBLE_EQ( 7.5623664855957, value );
 
+  std::vector<double> newVertex{10, 10, 10};
+  MDAL_M_addVertices( m, 1, newVertex.data() );
+  EXPECT_EQ( MDAL_LastStatus(), Err_IncompatibleMesh );
```

Comment:
add test that selafin mesh is not editable too,.

Reply:
hmm, this line tests that selafin mesh is not editable, I am not sure to understand. Do you want I add a `MDAL_Status` for non editable mesh?

Reply:
no problem